### PR TITLE
add active network to network selector

### PR DESCRIPTION
**Pull request by Jadessu**
### Describe your changes 
_Describe the purpose + content of these changes_
add active network to network selector
image

### Link the related issue
_Closes #0000_

### Checklist before requesting a review
- [x] Is this PR ready for merge? (Please convert to a draft PR otherwise)
- [x] I have performed a self-review of my code.
- [x] Did I request feedback from a team member prior to merge? 
- [x] Does my code following the style guide at `docs/CODING-STYLE.md`?

### CODE REVIEW

File: src/App/components/PageHeader/NetworkSelector/NetworkSelector.tsx [hunk 1]
```diff
@@ -64,15 +64,27 @@ export default function NetworkSelector(props: propsIF) {
...
+                    >
                         <img
                             src={chain.logoUrl}
                             alt={chain.displayName}
                             width='21px'
                             height='21px'
                             style={{ borderRadius: '50%' }}
                         />
-                        {chain.displayName}
+                        <p
```
Comment (cadezacharias-hype) [inline src/App/components/PageHeader/NetworkSelector/NetworkSelector.tsx:78]: Would prefer here a <Text> component from the common lib.  

`<Text color={chain.chainId === chainId && 'var(--accent1)'} />`

File: src/styled/Components/Header.ts [hunk 2]
```diff
@@ -534,7 +534,10 @@ export const NetworkItem = styled(motion.li)`
     }
 `;
 
-export const ChainNameStatus = styled.div`
+export const ChainNameStatus = styled.div<{ active: boolean }>`
+    display: flex;
+    flex-direction: row;
```
Comment (cadezacharias-hype) [inline src/styled/Components/Header.ts:539]: row is the default flex-direction, so not needed here. 


We could possible refactor this as a FlexContainer and just use all the props, but not crucial. would be something like 

```
<FlexContainer
alignItems='center'
padding='0.6rem 0'
...
/>

```

**Review by cadezacharias-hype — APPROVED** [review]
Couple tweaks on using styled components over the 'style' tag

**Review by benwolski — APPROVED** [review]
looks great!

File: src/styled/Components/Header.ts (hunk 2 above)
Comment (Jadessu) [reply src/styled/Components/Header.ts:539]: There are a few stylings being applied here that I would rather keep out of the jsx statement
`
    &:hover,
    &:focus-visible {
        filter: drop-shadow(0 0 5px var(--text1));
    }

    img {
        margin-right: 0.5em;
        vertical-align: middle;
    }

    border-bottom: ${(props) =>
        props.active ? '1px solid var(--accent1)' : ''};`

**Pull request merged**